src/methods/defining_methods/hard/q8/RentMethod.java

```diff
@@ -0,0 +1,27 @@
+package methods.defining_methods.hard.q8;
+
+/**
+ * [J] Write a Java Program that asks users for the cost of rent for their housing per month. Then
```

Comment:
Change question text to: 
`Write a Java Program that asks users for the cost of rent for their housing per month. Then write a Java method called rentCalc that accepts that value as a double and returns the cost per year for housing as a double. Your program should print the returned value in the main method. `